CMakeLists.txt

```diff
@@ -472,16 +472,20 @@ if(MNN_EVALUATION)
     add_subdirectory(tools/evaluation)
 endif()
 
-# schema generator
-if(WIN32)
-    add_custom_target( MNN_SCHEMA ALL
-        COMMAND powershell ${CMAKE_CURRENT_SOURCE_DIR}/schema/generate.ps1 -lazy
-    )
-else()
-    add_custom_target( MNN_SCHEMA ALL
-        COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/schema/generate.sh -lazy
-    )
+find_program(PYTHON_BIN python3)
```

Comment:
这里有什么不用``include(FindPythonInterp REQUIRED)`` 的理由吗

Reply:
我看了一下，这个FindPythonInterp。文档已经说明 Deprecated since version 3.12。不过我这里似乎使用find_program也有点欠佳。参考：https://cmake.org/cmake/help/latest/module/FindPythonInterp.html